phpunit.xml.dist

```diff
@@ -11,24 +11,26 @@
   processIsolation            = "false"
   stopOnFailure               = "false"
   syntaxCheck                 = "false"
-  bootstrap                   = "tests/bootstrap.php">
+  bootstrap                   = "vendor/autoload.php">
 
   <testsuites>
-    <testsuite name="Test suite">
-      <directory suffix="Test.php">./src/TreeHouse/KeystoneBundle/Tests</directory>
+    <testsuite>
+      <directory>./tests</directory>
     </testsuite>
   </testsuites>
 
-  <php>
-    <server name="KERNEL_DIR" value="./tests/Functional/app"/>
-  </php>
-
   <filter>
     <whitelist>
       <directory suffix=".php">./src</directory>
-      <exclude>
-        <directory>./src/TreeHouse/KeystoneBundle/Test</directory>
-      </exclude>
     </whitelist>
   </filter>
+
+  <php>
+    <server name="KERNEL_DIR" value="./tests/Functional/app"/>
+  </php>
+
+  <logging>
+    <log type="coverage-text" target="php://stdout"/>
+    <log type="coverage-html" target="build/coverage" charset="UTF-8" yui="true" highlight="true"/>
+  </logging>
```

Comment:
Again, from the skeleton.

Coverage is not logged at all by default. The HTML build is extra for when you want to inspect the coverage, a nice addition.